src/main/java/org/openhab/binding/zwave/internal/protocol/initialization/ZWaveNodeInitStageAdvancer.java

```diff
@@ -277,7 +277,6 @@ private boolean processTransaction(ZWaveMessagePayloadTransaction transaction, l
                 try {
                     Thread.sleep(backoff);
                 } catch (InterruptedException e) {
-                    e.printStackTrace();
```

Comment:
Should this have been replaced with a logger.error?

Reply:
No.